### ISSUE TITLE
Crashing when joining multiplayer servers

### ISSUE THREAD

**Issue by Amb0ww** [ImpactDevelopment/ImpactIssues#628]
**Issue:**
I usually keep crashing whenever I join a multiplayer server (Mineland) it gets stuck on "loading terrain", sometimes when I'm in the server it crashes again after a few minutes. The chances of crashing is higher when I play with an alt on cracked servers.

Bug?: Also whenever I press alt my game camera locks onto my previous camera position.

--------------------------

<details> 
  <summary><strong>Game output</strong></summary>
 
```
Logs: [23:16:29] [main/INFO]: Loading tweak class name clientapi.load.ClientTweaker
[23:16:29] [main/INFO]: Using primary tweak class name clientapi.load.ClientTweaker
[23:16:29] [main/INFO]: Calling tweak class clientapi.load.ClientTweaker
[23:16:29] [main/INFO]: Injecting into ClassLoader
[23:16:30] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.7.7 Source=file:/C:/Users/ambow/AppData/Roaming/.minecraft/libraries/org/spongepowered/mixin/0.7.7-SNAPSHOT/mixin-0.7.7-SNAPSHOT.jar Service=LaunchWrapper Env=UNKNOWN
[23:16:30] [main/INFO]: FML platform manager could not load class cpw.mods.fml.relauncher.CoreModManager. Proceeding without FML support.
[23:16:30] [main/INFO]: Initialized Mixin bootstrap
[23:16:30] [main/INFO]: Compatibility level set to JAVA_8
[23:16:30] [main/ERROR]: Mixin config mixins.capi.json does not specify "minVersion" property
[23:16:30] [main/INFO]: Loaded ClientAPI mixin configuration
[23:16:30] [main/ERROR]: Mixin config mixins.client.json does not specify "minVersion" property
[23:16:30] [main/INFO]: Loaded Client mixin configuration
[23:16:30] [main/INFO]: Loading tweak class name org.spongepowered.asm.mixin.EnvironmentStateTweaker
[23:16:30] [main/INFO]: Calling tweak class org.spongepowered.asm.mixin.EnvironmentStateTweaker
[23:16:30] [main/INFO]: Launching wrapped minecraft {net.minecraft.client.main.Main}
[23:16:32] [main/INFO]: Setting user: Amboww
[23:16:34] [main/WARN]: Skipping bad option: lastServer:
[23:16:34] [main/INFO]: LWJGL Version: 2.9.4
[23:16:34] [main/INFO]: Reloading ResourceManager: Default, Faithful+1.12.2-rv4.zip
[23:16:35] [Sound Library Loader/INFO]: Starting up SoundSystem...
[23:16:35] [Thread-3/INFO]: Initializing LWJGL OpenAL
[23:16:35] [Thread-3/INFO]: (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
[23:16:35] [Thread-3/INFO]: OpenAL initialized.
[23:16:35] [Sound Library Loader/INFO]: Sound engine started
[23:16:38] [main/INFO]: Created: 2048x1024 textures-atlas
[23:16:39] [main/INFO]: Narrator library for x64 successfully loaded
[23:16:42] [main/INFO]: Loading Fonts
[23:16:43] [main/WARN]: Checking for Updates
[23:16:43] [main/INFO]: Loading Users
[23:16:44] [main/INFO]: Initialized 106 mods!
[23:16:44] [main/FATAL]: Unable to load spam config file: Spam config file is in invalid format
[23:16:44] [main/ERROR]: ########## GL ERROR ##########
[23:16:44] [main/ERROR]: @ Pre render
[23:16:44] [main/ERROR]: 1282: Invalid operation
[23:16:55] [Server thread/INFO]: Starting integrated minecraft server version 1.12.2
[23:16:55] [Server thread/INFO]: Generating keypair
[23:16:56] [Server thread/INFO]: Loaded 488 advancements
[23:16:56] [Server thread/INFO]: Preparing start region for level 0
[23:16:57] [Server thread/INFO]: Preparing spawn area: 76%
[23:16:57] [Server thread/INFO]: Changing view distance to 16, from 10
[23:16:59] [Server thread/INFO]: Amboww[local:E:a0b0eee6] logged in with entity id 23 at (266.312497825601, 72.99323530006211, -261.6611349797497)
[23:16:59] [Server thread/INFO]: Amboww joined the game
[23:17:01] [Server thread/WARN]: Can't keep up! Did the system time change, or is the server overloaded? Running 2770ms behind, skipping 55 tick(s)
[23:17:02] [main/INFO]: Loaded 18 advancements
[23:17:10] [Server thread/INFO]: Saving and pausing game...
[23:17:10] [Server thread/INFO]: Saving chunks for level 'Fierst'/overworld
[23:17:11] [Server thread/INFO]: Saving chunks for level 'Fierst'/the_nether
[23:17:11] [Server thread/INFO]: Saving chunks for level 'Fierst'/the_end
[23:17:15] [Server thread/INFO]: Saving and pausing game...
[23:17:15] [Server thread/INFO]: Saving chunks for level 'Fierst'/overworld
[23:17:15] [Server thread/INFO]: Saving chunks for level 'Fierst'/the_nether
[23:17:15] [Server thread/INFO]: Saving chunks for level 'Fierst'/the_end
[23:18:17] [Server thread/INFO]: Saving and pausing game...
[23:18:17] [Server thread/INFO]: Saving chunks for level 'Fierst'/overworld
[23:18:17] [Server thread/INFO]: Saving chunks for level 'Fierst'/the_nether
[23:18:17] [Server thread/INFO]: Saving chunks for level 'Fierst'/the_end
[23:19:07] [Server thread/INFO]: Saving and pausing game...
[23:19:07] [Server thread/INFO]: Saving chunks for level 'Fierst'/overworld
[23:19:07] [Server thread/INFO]: Saving chunks for level 'Fierst'/the_nether
[23:19:07] [Server thread/INFO]: Saving chunks for level 'Fierst'/the_end

```

</details>

--------------------------

<details> 
  <summary><strong>Crash report</strong></summary>
 
```
Crash Report: ---- Minecraft Crash Report ----

Time: 6/22/18 11:42 PM
Description: Unexpected error

java.util.ConcurrentModificationException
	at java.util.ArrayList.forEach(ArrayList.java:1252)
	at me.zero.clarinet.w.lambda$new$2(w.java)
	at me.zero.clarinet.w$$Lambda$118/1374950118.invoke(Unknown Source)
	at me.zero.alpine.listener.Listener.invoke(Listener.java:77)
	at me.zero.alpine.EventManager.lambda$post$6(EventManager.java:98)
	at me.zero.alpine.EventManager$$Lambda$95/1594964071.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at me.zero.alpine.EventManager.post(EventManager.java:98)
	at me.zero.client.api.event.CAPIEventManager.post(CAPIEventManager.java:34)
	at me.zero.client.api.event.handle.ClientHandler.lambda$new$7(ClientHandler.java:85)
	at me.zero.client.api.event.handle.ClientHandler$$Lambda$49/1034461954.invoke(Unknown Source)
	at me.zero.alpine.listener.Listener.invoke(Listener.java:77)
	at me.zero.alpine.EventManager.lambda$post$6(EventManager.java:98)
	at me.zero.alpine.EventManager$$Lambda$95/1594964071.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at me.zero.alpine.EventManager.post(EventManager.java:98)
	at me.zero.client.api.event.CAPIEventManager.post(CAPIEventManager.java:34)
	at rl.handler$startSection$zzl000(SourceFile:539)
	at rl.a(SourceFile)
	at rl.c(SourceFile:150)
	at buq.a(SourceFile:1354)
	at buq.b(SourceFile:1188)
	at buq.a(SourceFile:1033)
	at bib.az(SourceFile:1023)
	at bib.a(SourceFile:419)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at java.util.ArrayList.forEach(ArrayList.java:1252)
	at me.zero.clarinet.w.lambda$new$2(w.java)
	at me.zero.clarinet.w$$Lambda$118/1374950118.invoke(Unknown Source)
	at me.zero.alpine.listener.Listener.invoke(Listener.java:77)
	at me.zero.alpine.EventManager.lambda$post$6(EventManager.java:98)
	at me.zero.alpine.EventManager$$Lambda$95/1594964071.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at me.zero.alpine.EventManager.post(EventManager.java:98)
	at me.zero.client.api.event.CAPIEventManager.post(CAPIEventManager.java:34)
	at me.zero.client.api.event.handle.ClientHandler.lambda$new$7(ClientHandler.java:85)
	at me.zero.client.api.event.handle.ClientHandler$$Lambda$49/1034461954.invoke(Unknown Source)
	at me.zero.alpine.listener.Listener.invoke(Listener.java:77)
	at me.zero.alpine.EventManager.lambda$post$6(EventManager.java:98)
	at me.zero.alpine.EventManager$$Lambda$95/1594964071.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at me.zero.alpine.EventManager.post(EventManager.java:98)
	at me.zero.client.api.event.CAPIEventManager.post(CAPIEventManager.java:34)
	at rl.handler$startSection$zzl000(SourceFile:539)
	at rl.a(SourceFile)
	at rl.c(SourceFile:150)
	at buq.a(SourceFile:1354)
	at buq.b(SourceFile:1188)

-- Affected level --
Details:
	Level name: MpServer
	All players: 12 total; [b['Jublonkski'/690756, l='MpServer', x=8.50, y=65.00, z=8.50], bue['Ernes'/661376, l='MpServer', x=-99.82, y=75.00, z=393.11], bue['MCSuperBoy'/690651, l='MpServer', x=-98.02, y=75.00, z=395.20], bue['BattleAxeMC'/689887, l='MpServer', x=-107.70, y=76.00, z=398.31], bue['alex_bg'/631521, l='MpServer', x=-99.82, y=75.00, z=395.05], bue['JohnynPizza'/558968, l='MpServer', x=-99.90, y=75.00, z=394.84], bue['In_The_Dark'/686168, l='MpServer', x=-106.68, y=75.00, z=402.46], bue['Liuciuji_PLY'/689891, l='MpServer', x=-102.72, y=75.00, z=404.84], bue['rumipro'/689664, l='MpServer', x=-75.48, y=75.00, z=387.54], bue['Kahama'/688916, l='MpServer', x=-75.23, y=75.00, z=393.44], bue['XproOyuncuTR'/690653, l='MpServer', x=-73.74, y=75.10, z=385.46], bue['TheSage14'/689017, l='MpServer', x=-78.87, y=75.00, z=390.24]]
	Chunk stats: MultiplayerChunkCache: 270, 270
	Level seed: 0
	Level generator: ID 01 - flat, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 131 game time, 131 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: adventure (ID 2). Hardcore: false. Cheats: false
	Forced entities: 38 total; [bue['Liuciuji_PLY'/689891, l='MpServer', x=-102.72, y=75.00, z=404.84], acl['§fWay to §a[SkyWars]'/690444, l='MpServer', x=-92.88, y=70.00, z=378.98], acl['§fWay to §a[BedWars]'/689228, l='MpServer', x=-87.45, y=70.00, z=380.91], acl['§fWay to §a[BedWars]'/690125, l='MpServer', x=-80.47, y=73.50, z=381.26], bue['rumipro'/689664, l='MpServer', x=-75.48, y=75.00, z=387.54], acl['§r§fWay to §r§c§lIsland'/689346, l='MpServer', x=-114.75, y=75.00, z=415.38], bue['Kahama'/688916, l='MpServer', x=-75.23, y=75.00, z=393.44], bue['XproOyuncuTR'/690653, l='MpServer', x=-73.74, y=75.10, z=385.46], bue['Ernes'/661376, l='MpServer', x=-99.82, y=75.00, z=393.11], acl['§fWay to §a[BedWars]'/689344, l='MpServer', x=-83.13, y=72.00, z=380.78], bue['rumipro'/689664, l='MpServer', x=-75.48, y=75.00, z=387.54], bue['TheSage14'/689017, l='MpServer', x=-78.87, y=75.00, z=390.24], acl['§r§fWay to §r§c§lIsland'/689345, l='MpServer', x=-113.83, y=75.00, z=412.00], bue['MCSuperBoy'/690651, l='MpServer', x=-98.02, y=75.00, z=395.20], bue['In_The_Dark'/686168, l='MpServer', x=-106.68, y=75.00, z=402.46], bue['BattleAxeMC'/689887, l='MpServer', x=-107.70, y=76.00, z=398.31], bue['XproOyuncuTR'/690653, l='MpServer', x=-73.74, y=75.10, z=385.46], acl['§fWay to §a[SkyWars]'/689555, l='MpServer', x=-100.83, y=72.00, z=380.91], acl['§fWay to §a[SkyWars]'/689558, l='MpServer', x=-96.64, y=72.00, z=380.38], bue['Kahama'/688916, l='MpServer', x=-75.23, y=75.00, z=393.44], bue['alex_bg'/631521, l='MpServer', x=-99.82, y=75.00, z=395.05], bue['Liuciuji_PLY'/689891, l='MpServer', x=-102.72, y=75.00, z=404.84], bue['JohnynPizza'/558968, l='MpServer', x=-99.90, y=75.00, z=394.84], acl['§r§fWay to §r§c§lIsland'/689338, l='MpServer', x=-107.50, y=75.00, z=406.00], acl['§fWay to §a[BedWars]'/690234, l='MpServer', x=-70.72, y=75.00, z=383.78], acl['§r§fWay to §r§c§lIsland'/689339, l='MpServer', x=-111.11, y=75.00, z=409.31], acl['§fWay to §a[BedWars]'/690235, l='MpServer', x=-74.37, y=75.00, z=381.59], acl['§fWay to §a[SkyWars]'/689336, l='MpServer', x=-100.88, y=73.00, z=385.32], b['Jublonkski'/690756, l='MpServer', x=8.50, y=65.00, z=8.50], acl['§fWay to §a[SkyWars]'/689337, l='MpServer', x=-101.14, y=74.50, z=390.30], bue['TheSage14'/689017, l='MpServer', x=-78.87, y=75.00, z=390.24], bue['Ernes'/661376, l='MpServer', x=-99.82, y=75.00, z=393.11], bue['MCSuperBoy'/690651, l='MpServer', x=-98.02, y=75.00, z=395.20], bue['BattleAxeMC'/689887, l='MpServer', x=-107.70, y=76.00, z=398.31], bue['alex_bg'/631521, l='MpServer', x=-99.82, y=75.00, z=395.05], bue['JohnynPizza'/558968, l='MpServer', x=-99.90, y=75.00, z=394.84], acl['§r§fWay to §r§c§lIsland'/689783, l='MpServer', x=-117.38, y=75.00, z=420.44], bue['In_The_Dark'/686168, l='MpServer', x=-106.68, y=75.00, z=402.46]]
	Retry entities: 0 total; []
	Server brand: BotFilter (unknown) <- Paper
	Server type: Non-integrated multiplayer server
Stacktrace:
	at bsb.a(SourceFile:366)
	at bib.b(SourceFile:2587)
	at bib.a(SourceFile:433)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 297312168 bytes (283 MB) / 654311424 bytes (624 MB) up to 1073741824 bytes (1024 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=16M
	IntCache: cache: 1, tcache: 1, allocated: 12, tallocated: 94
	Launched Version: 1.12.2-Impact_4.1
	LWJGL: 2.9.4
	OpenGL: Intel(R) HD Graphics 630 GL version 4.5.0 - Build 23.20.16.4939, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: Faithful+1.12.2-rv4.zip
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 8x Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
```

</details>

--------------------------

<details> 
  <summary><strong>Additional Client Info</strong></summary><br>
 
- Impact Version: 4.2 for 1.12.2
- Using Optifine? No
- Enabled mods: https://gyazo.com/0efe73e91a43fc020ddf6a7f81fc3f38
- Operating System: N/A

</details>

**Comment by LordQuadrato** [comment]
I edited your issue a little bit for a clear layout.

**Can you please include you crash report. You can find it in `.minecraft/crash-reports`**

**Comment by Amb0ww** [comment]
Crash Logs: ---- Minecraft Crash Report ----
// You're mean.

Time: 6/22/18 11:42 PM
Description: Unexpected error

java.util.ConcurrentModificationException
	at java.util.ArrayList.forEach(ArrayList.java:1252)
	at me.zero.clarinet.w.lambda$new$2(w.java)
	at me.zero.clarinet.w$$Lambda$118/1374950118.invoke(Unknown Source)
	at me.zero.alpine.listener.Listener.invoke(Listener.java:77)
	at me.zero.alpine.EventManager.lambda$post$6(EventManager.java:98)
	at me.zero.alpine.EventManager$$Lambda$95/1594964071.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at me.zero.alpine.EventManager.post(EventManager.java:98)
	at me.zero.client.api.event.CAPIEventManager.post(CAPIEventManager.java:34)
	at me.zero.client.api.event.handle.ClientHandler.lambda$new$7(ClientHandler.java:85)
	at me.zero.client.api.event.handle.ClientHandler$$Lambda$49/1034461954.invoke(Unknown Source)
	at me.zero.alpine.listener.Listener.invoke(Listener.java:77)
	at me.zero.alpine.EventManager.lambda$post$6(EventManager.java:98)
	at me.zero.alpine.EventManager$$Lambda$95/1594964071.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at me.zero.alpine.EventManager.post(EventManager.java:98)
	at me.zero.client.api.event.CAPIEventManager.post(CAPIEventManager.java:34)
	at rl.handler$startSection$zzl000(SourceFile:539)
	at rl.a(SourceFile)
	at rl.c(SourceFile:150)
	at buq.a(SourceFile:1354)
	at buq.b(SourceFile:1188)
	at buq.a(SourceFile:1033)
	at bib.az(SourceFile:1023)
	at bib.a(SourceFile:419)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at java.util.ArrayList.forEach(ArrayList.java:1252)
	at me.zero.clarinet.w.lambda$new$2(w.java)
	at me.zero.clarinet.w$$Lambda$118/1374950118.invoke(Unknown Source)
	at me.zero.alpine.listener.Listener.invoke(Listener.java:77)
	at me.zero.alpine.EventManager.lambda$post$6(EventManager.java:98)
	at me.zero.alpine.EventManager$$Lambda$95/1594964071.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at me.zero.alpine.EventManager.post(EventManager.java:98)
	at me.zero.client.api.event.CAPIEventManager.post(CAPIEventManager.java:34)
	at me.zero.client.api.event.handle.ClientHandler.lambda$new$7(ClientHandler.java:85)
	at me.zero.client.api.event.handle.ClientHandler$$Lambda$49/1034461954.invoke(Unknown Source)
	at me.zero.alpine.listener.Listener.invoke(Listener.java:77)
	at me.zero.alpine.EventManager.lambda$post$6(EventManager.java:98)
	at me.zero.alpine.EventManager$$Lambda$95/1594964071.accept(Unknown Source)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at me.zero.alpine.EventManager.post(EventManager.java:98)
	at me.zero.client.api.event.CAPIEventManager.post(CAPIEventManager.java:34)
	at rl.handler$startSection$zzl000(SourceFile:539)
	at rl.a(SourceFile)
	at rl.c(SourceFile:150)
	at buq.a(SourceFile:1354)
	at buq.b(SourceFile:1188)

-- Affected level --
Details:
	Level name: MpServer
	All players: 12 total; [b['Jublonkski'/690756, l='MpServer', x=8.50, y=65.00, z=8.50], bue['Ernes'/661376, l='MpServer', x=-99.82, y=75.00, z=393.11], bue['MCSuperBoy'/690651, l='MpServer', x=-98.02, y=75.00, z=395.20], bue['BattleAxeMC'/689887, l='MpServer', x=-107.70, y=76.00, z=398.31], bue['alex_bg'/631521, l='MpServer', x=-99.82, y=75.00, z=395.05], bue['JohnynPizza'/558968, l='MpServer', x=-99.90, y=75.00, z=394.84], bue['In_The_Dark'/686168, l='MpServer', x=-106.68, y=75.00, z=402.46], bue['Liuciuji_PLY'/689891, l='MpServer', x=-102.72, y=75.00, z=404.84], bue['rumipro'/689664, l='MpServer', x=-75.48, y=75.00, z=387.54], bue['Kahama'/688916, l='MpServer', x=-75.23, y=75.00, z=393.44], bue['XproOyuncuTR'/690653, l='MpServer', x=-73.74, y=75.10, z=385.46], bue['TheSage14'/689017, l='MpServer', x=-78.87, y=75.00, z=390.24]]
	Chunk stats: MultiplayerChunkCache: 270, 270
	Level seed: 0
	Level generator: ID 01 - flat, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 131 game time, 131 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: adventure (ID 2). Hardcore: false. Cheats: false
	Forced entities: 38 total; [bue['Liuciuji_PLY'/689891, l='MpServer', x=-102.72, y=75.00, z=404.84], acl['§fWay to §a[SkyWars]'/690444, l='MpServer', x=-92.88, y=70.00, z=378.98], acl['§fWay to §a[BedWars]'/689228, l='MpServer', x=-87.45, y=70.00, z=380.91], acl['§fWay to §a[BedWars]'/690125, l='MpServer', x=-80.47, y=73.50, z=381.26], bue['rumipro'/689664, l='MpServer', x=-75.48, y=75.00, z=387.54], acl['§r§fWay to §r§c§lIsland'/689346, l='MpServer', x=-114.75, y=75.00, z=415.38], bue['Kahama'/688916, l='MpServer', x=-75.23, y=75.00, z=393.44], bue['XproOyuncuTR'/690653, l='MpServer', x=-73.74, y=75.10, z=385.46], bue['Ernes'/661376, l='MpServer', x=-99.82, y=75.00, z=393.11], acl['§fWay to §a[BedWars]'/689344, l='MpServer', x=-83.13, y=72.00, z=380.78], bue['rumipro'/689664, l='MpServer', x=-75.48, y=75.00, z=387.54], bue['TheSage14'/689017, l='MpServer', x=-78.87, y=75.00, z=390.24], acl['§r§fWay to §r§c§lIsland'/689345, l='MpServer', x=-113.83, y=75.00, z=412.00], bue['MCSuperBoy'/690651, l='MpServer', x=-98.02, y=75.00, z=395.20], bue['In_The_Dark'/686168, l='MpServer', x=-106.68, y=75.00, z=402.46], bue['BattleAxeMC'/689887, l='MpServer', x=-107.70, y=76.00, z=398.31], bue['XproOyuncuTR'/690653, l='MpServer', x=-73.74, y=75.10, z=385.46], acl['§fWay to §a[SkyWars]'/689555, l='MpServer', x=-100.83, y=72.00, z=380.91], acl['§fWay to §a[SkyWars]'/689558, l='MpServer', x=-96.64, y=72.00, z=380.38], bue['Kahama'/688916, l='MpServer', x=-75.23, y=75.00, z=393.44], bue['alex_bg'/631521, l='MpServer', x=-99.82, y=75.00, z=395.05], bue['Liuciuji_PLY'/689891, l='MpServer', x=-102.72, y=75.00, z=404.84], bue['JohnynPizza'/558968, l='MpServer', x=-99.90, y=75.00, z=394.84], acl['§r§fWay to §r§c§lIsland'/689338, l='MpServer', x=-107.50, y=75.00, z=406.00], acl['§fWay to §a[BedWars]'/690234, l='MpServer', x=-70.72, y=75.00, z=383.78], acl['§r§fWay to §r§c§lIsland'/689339, l='MpServer', x=-111.11, y=75.00, z=409.31], acl['§fWay to §a[BedWars]'/690235, l='MpServer', x=-74.37, y=75.00, z=381.59], acl['§fWay to §a[SkyWars]'/689336, l='MpServer', x=-100.88, y=73.00, z=385.32], b['Jublonkski'/690756, l='MpServer', x=8.50, y=65.00, z=8.50], acl['§fWay to §a[SkyWars]'/689337, l='MpServer', x=-101.14, y=74.50, z=390.30], bue['TheSage14'/689017, l='MpServer', x=-78.87, y=75.00, z=390.24], bue['Ernes'/661376, l='MpServer', x=-99.82, y=75.00, z=393.11], bue['MCSuperBoy'/690651, l='MpServer', x=-98.02, y=75.00, z=395.20], bue['BattleAxeMC'/689887, l='MpServer', x=-107.70, y=76.00, z=398.31], bue['alex_bg'/631521, l='MpServer', x=-99.82, y=75.00, z=395.05], bue['JohnynPizza'/558968, l='MpServer', x=-99.90, y=75.00, z=394.84], acl['§r§fWay to §r§c§lIsland'/689783, l='MpServer', x=-117.38, y=75.00, z=420.44], bue['In_The_Dark'/686168, l='MpServer', x=-106.68, y=75.00, z=402.46]]
	Retry entities: 0 total; []
	Server brand: BotFilter (unknown) <- Paper
	Server type: Non-integrated multiplayer server
Stacktrace:
	at bsb.a(SourceFile:366)
	at bib.b(SourceFile:2587)
	at bib.a(SourceFile:433)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 297312168 bytes (283 MB) / 654311424 bytes (624 MB) up to 1073741824 bytes (1024 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=16M
	IntCache: cache: 1, tcache: 1, allocated: 12, tallocated: 94
	Launched Version: 1.12.2-Impact_4.1
	LWJGL: 2.9.4
	OpenGL: Intel(R) HD Graphics 630 GL version 4.5.0 - Build 23.20.16.4939, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: Faithful+1.12.2-rv4.zip
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 8x Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz

**Comment by ZeroMemes** [comment]
Can you reproduce this on the latest Impact version? This is on 4.1 so it's ~11 months old.
